### PR TITLE
docs: add README for Airflow DAG sample, update code formatting in other docs

### PR DESCRIPTION
**Pull request by helensilva14**
Add a README for the Airflow DAG sample explicitly adding a warning information to avoid confusion regarding the configuration file/params when new users want to try it.

Took the chance to update the code formatting in other docs files.

### CODE REVIEW

**Review by Raniksingh — COMMENTED** [review]
how about to add one example here like source_conn in json and source in yaml?

**Comment by helensilva14** [comment]
> how about to add one example here like source_conn in json and source in yaml?

Added!

File: samples/airflow/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+The Airflow DAG code expects the raw config JSON only, 
+it does not support a JSON format that was generated by converting an existing
+YAML config file.
+
+Parameters from YAML format are slightly different from the JSON config,
+there is an example for Cloud Run [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/tree/develop/samples/run#test-cloud-run-endpoint) 
+that shows the expected JSON config in the `'data'` variable.
+
+For example, check the following YAML config part.
```
Comment (nehanene15) [inline samples/airflow/README.md]:
```suggestion
For example, the following YAML config is equivalent to the JSON config below. 
```

File: samples/airflow/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    target_column: null
+    type: count
+...
+```
+
+This YAML is equivalent to the JSON part below, where the source param is written as `source_conn`.
+
+```json
+{
+    "source_conn": BQ_CONN,
```
Comment (nehanene15) [inline samples/airflow/README.md:49]: Some of this text on this line and the ones below are highlighted.. is there a way to fix that?

File: samples/airflow/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+
+### Warning 
+
+The Airflow DAG code expects the raw config JSON only, 
+it does not support a JSON format that was generated by converting an existing
+YAML config file.
+
+Parameters from YAML format are slightly different from the JSON config,
+there is an example for Cloud Run [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/tree/develop/samples/run#test-cloud-run-endpoint)
```
Comment (nehanene15) [inline samples/airflow/README.md]: Let's add some more detail like so:

Parameters in a typical YAML config file are slightly different from the raw JSON config, which is generated after DVT parses the YAML. The [build_config_manager() method ](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/data_validation/config_manager.py#L425) generates the JSON config and should be used as a reference.

Our Cloud Run sample also expects a raw JSON config in the `'data'` variable shown [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/tree/develop/samples/run#test-cloud-run-endpoint)

File: samples/airflow/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+
+### Warning 
+
+The Airflow DAG code expects the raw config JSON only, 
+it does not support a JSON format that was generated by converting an existing
```
Comment (nehanene15) [inline samples/airflow/README.md]: We can make lines 4-6 more concise:

```suggestion
The Airflow DAG expects the raw config JSON which is not the same as a YAML config converted to JSON.
```

File: samples/airflow/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+
+### Warning
```
Comment (nehanene15) [inline samples/airflow/README.md]: Let's make this README similar to the [Cloud Run](https://github.com/GoogleCloudPlatform/professional-services-data-validator/tree/develop/samples/run) or [Cloud Functions](https://github.com/GoogleCloudPlatform/professional-services-data-validator/tree/develop/samples/functions) samples. We should add a title 'Data Validation on Airflow' and explain how to deploy this on Airflow/Composer. 

Under the high level overview we can include this 'Warning' section. That way the README is a more complete documentation for the Airflow sample code.

**Comment by helensilva14** [comment]
/gcbrun.

File: samples/airflow/README.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+There is no specific operator for it at the moment, so the DAG uses the
+main structures of the library itself.
+
+The input is a JSON data representation of the DVT run configuration. 
+
+By default, the DAG will output the results to BigQuery as a result handler.
+
+### Requirements to run the DAG:
+- Pre-existing Airflow environment created with Public IP (Private environment disabled)
+- Create an Airflow variable called `gcp_project with the GCP Project ID
```
Comment (nehanene15) [inline samples/airflow/README.md]:
```suggestion
- Create an Airflow variable called `gcp_project with the GCP Project ID`
```

File: samples/airflow/README.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+### Instructions
+
+1. Download the Python file with the DAG
+2. Update the JSON configuration to your use case (source, target tables, etc)
+3. Upload it to the DAGs folder of your Airflow environment
```
Comment (nehanene15) [inline samples/airflow/README.md]:
```suggestion
3. Upload it to the DAGs folder in your Airflow environment
```

File: samples/airflow/README.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+By default, the DAG will output the results to BigQuery as a result handler.
+
+### Requirements to run the DAG:
+- Pre-existing Airflow environment created with Public IP (Private environment disabled)
+- Create an Airflow variable called `gcp_project with the GCP Project ID
+
+### Instructions
+
+1. Download the Python file with the DAG
+2. Update the JSON configuration to your use case (source, target tables, etc)
```
Comment (nehanene15) [inline samples/airflow/README.md]:
```suggestion
2. Update the JSON configuration for your use case (update table names, etc.)
```

File: samples/airflow/README.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+By default, the DAG will output the results to BigQuery as a result handler.
+
+### Requirements to run the DAG:
+- Pre-existing Airflow environment created with Public IP (Private environment disabled)
+- Create an Airflow variable called `gcp_project with the GCP Project ID
+
+### Instructions
+
+1. Download the Python file with the DAG
```
Comment (nehanene15) [inline samples/airflow/README.md]:
```suggestion
1. Download the DAG file in this directory
```

**Review by nehanene15 — COMMENTED** [review]
LGTM besides small formatting changes!

**Review by nehanene15 — APPROVED** [review]

**Pull request merged**